source/extensions/filters/http/common/compressor/compressor.h

```diff
@@ -0,0 +1,167 @@
+#pragma once
+
+#include "envoy/compressor/compressor.h"
+#include "envoy/config/filter/http/compressor/v2/compressor.pb.h"
+#include "envoy/http/filter.h"
+#include "envoy/runtime/runtime.h"
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats_macros.h"
+
+#include "common/buffer/buffer_impl.h"
+#include "common/common/macros.h"
+#include "common/common/utility.h"
+#include "common/protobuf/protobuf.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Common {
+namespace Compressors {
+
+/**
+ * All compressor filter stats. @see stats_macros.h
+ * "total_uncompressed_bytes" only includes bytes
+ * from requests that were marked for compression.
+ * If the request was not marked for compression,
+ * the filter increments "not_compressed", but does
+ * not add to "total_uncompressed_bytes". This way,
+ * the user can measure the memory performance of the
+ * compression.
+ * "header_gzip" is specific to the gzip filter and is
+ * deprecated since it duplicates "header_compressor_used".
+ */
```

Comment:
This is super nit and I know this is not yours, but we have `ColumnLimit: 100`, can we expand it a little bit?